module/src/ranking_challenge/survey.py

```diff
@@ -122,6 +130,70 @@ class SurveyResponse(BaseModel):
         description="Think of the past two weeks. How much time did you spend on social media, on average, per day?"
     )
 
+    feed_lean: SocmedLean = Field(

```

Comment:
I think we may want these fields to be optional. Like I am not sure if we are guaranteed to always get complete surveys, or if people can skip questions/submit partial answers.